core/model/PlatformResult.h

```diff
@@ -21,17 +21,43 @@
 
 namespace wolkabout
 {
-enum class PlatformResult
+class PlatformResult
 {
-    OK = 0,
-    ERROR_GATEWAY_NOT_FOUND,
-    ERROR_KEY_MISSING,
-    ERROR_NOT_A_GATEWAY,
-    ERROR_DEVICE_NOT_FOUND,
-    ERROR_INVALID_DEVICE
-};
+public:
+    enum class Code
+    {
+        OK = 0,
+        ERROR_GATEWAY_NOT_FOUND,
+        ERROR_DEVICE_NOT_FOUND,
+        ERROR_SUBDEVICE_MANAGEMENT_FORBIDDEN,
+        ERROR_MAXIMUM_NUMBER_OF_DEVICES_EXCEEDED,
+        ERROR_NOT_A_GATEWAY,
+        ERROR_INVALID_DEVICE,
+        ERROR_KEY_CONFLICT,
```

Comment:
i know there is already a bug/task to remove this one and replace it with something that is correct. not sure what is the version when it will be merged, but i can check.
therefor, like I already suggested. drop the list.

Reply:
This is left to be in line with the documentation. Regardless of the list we shall accept any error strings returned by platform